src/pres/MainContainer.js

```diff
@@ -38,28 +39,28 @@ export default class MainContainer extends React.Component {
   constructor(props){
     super(props)
 
-    let urlVariant = new URLSearchParams(window.location.search).get("variant")
-    let selectedVariant = urlVariant || Constants.VARIANT_STANDARD
-    this.chess = chessLogic(selectedVariant)
+    const urlSettings = this.getSettingsFromUrlParams()
```

Comment:
I'm not super sure what to call `urlSettings`. Most obvious would just be `settings`, but I see there's something called that already, which I think is for settings stored in a cookie.

Maybe:

- Rename `settings` to `persistentSettings`, AND
- Rename `urlSettings` to `settings`

OR

- Merge `urlSettings` and `settings` together in some way, and call it just `settings`

What do you think?

Reply:
Oh, also, there is one place where urlSettings and settings collide -- player color https://github.com/openingtree/openingtree/pull/273/files#r594764448